app/views/metadata_presenter/page/checkanswers.html.erb

```diff
@@ -35,25 +35,36 @@
     <%= form_for @page, url: reserved_submissions_path do |f| %>
       <div data-block-id="page.checkanswers.answers" data-block-type="answers">
         <dl class="fb-block fb-block-answers govuk-summary-list">
-          <% page_answers_presenters.each do |page_answers_presenter| %>
-            <div class="govuk-summary-list__row">
-              <dt class="govuk-summary-list__key">
-                <%= page_answers_presenter.humanised_title %>
-              </dt>
+          <% pages_presenters.each do |page_answers_presenters| %>
+            <% page_answers_presenters.each_with_index do |page_answers_presenter, index| %>
 
-              <dd class="govuk-summary-list__value">
-                <%= page_answers_presenter.answer %>
-              </dd>
-              <dd class="govuk-summary-list__actions">
-              <%= link_to(
-                    change_answer_path(url: page_answers_presenter.url),
-                    class: 'govuk-link'
-                  ) do %>
-                     Change<span class="govuk-visually-hidden"> Your answer for <%= page_answers_presenter.humanised_title %></span>
-                <% end %>
-                </a>
-              </dd>
-            </div>
+              <% if page_answers_presenter.multiple_questions? && index == 0 %>
```

Comment:
Can we create and move this condition to page_answers_presenter.display_heading?